newsletter-september-2021.rst

```diff
@@ -0,0 +1,101 @@
+.. post:: September 2, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in September.
+
+Read the Docs newsletter - September 2021
+=========================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We have finally published
+  :doc:`the first release candidate of version 1.0.0 of our Sphinx theme </theme-release-100rc1>`,
+  which adds support for recent versions of Sphinx and docutils among other things,
+  and announced our future plans for it.
+  Check out the linked blog post to know more.
+- The :doc:`first part of the new Read the Docs tutorial <readthedocs:tutorial/index>`,
+  which we wrote as part of :doc:`our CZI grant </czi-grant-announcement>`, is online!
+  We hope that it serves as a better starting point for people seeking to learn how to use our platform.
+
+New features
+------------
+
+- We added the possibility for users to `remove themselves from a
+  project <https://github.com/readthedocs/readthedocs.org/pull/8384>`_
+  without having to ask an owner to remove them
+  (as long as they are not the only owner of the project).
+- We fixed `a small usability issue with our search-as-you-type
+  box <https://github.com/readthedocs/readthedocs-sphinx-search/pull/93>`_,
+  that ignored the first typed characters under certain circumstances.
+- Commercial users can now `share specific versions of their
+  projects <https://docs.readthedocs.io/en/stable/commercial/sharing.html>`_
+  and `log out from the flyout
+  menu <https://docs.readthedocs.io/en/stable/versions.html#logging-out>`_.
```

Comment:
I'd put this higher, it seems like the most useful feature.

Reply:
Done!